CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ### Added
 - `AuroraPostgresLsnReplicaConsistency`
 
+### Fixed
+- Implemented missing methods in `DualConnection`
```

Comment:
NIT: I wonder why you adding it under `1.1.0` version ? IMHO you should add it under the new version, in _mark commit ( to update changelog and readme) _

Reply:
Good catch, this should be under `Unreleased`